book/chapter-6/6.6-create-essentials.sh

```diff
@@ -23,7 +23,7 @@ ln -sv /proc/self/mounts /etc/mtab || true
 
 # configure root
 cat > /etc/passwd << "EOF"
-root::0:0:root:/root:/bin/bash
+root:x:0:0:root:/root:/bin/bash
```

Comment:
Yeah! Now it should work with empty password too (if not set). But, I still think that setting a password is a better option.

Reply:
I think there is no need to worry about this, whoever needs it will set the password himself, after all, this is an ISO image for experiments and additional difficulties will confuse people.

Reply:
Makes sense